Makefile

```diff
@@ -45,8 +45,7 @@ PACKAGE_PROVIDES ?= `grep PACKAGE_PROVIDES $(KONG_SOURCE_LOCATION)/.requirements
 PACKAGE_REPLACES ?= `grep PACKAGE_REPLACES $(KONG_SOURCE_LOCATION)/.requirements | awk -F"=" '{print $$2}'`
 DOCKER_RELEASE_REPOSITORY?="kong/kong"
 
-# This logic should mirror the kong-build-tools equivalent
-KONG_VERSION?=`$(KONG_SOURCE_LOCATION)/distribution/grep-kong-version.sh`
+KONG_VERSION?=`./grep-kong-version.sh $(KONG_SOURCE_LOCATION)`
```

Comment:
Since this should only live in this repo from now on, WDYT about renaming this to `detect-kong-version` or something like that?